src/clarity-angular/layout/tabs/tab-content.spec.ts

```diff
@@ -25,18 +26,18 @@ describe("TabContent", () => {
     beforeEach(() => {
         TestBed.configureTestingModule({
             imports: [ClrTabsModule],
-            declarations: [TestComponent]
+            declarations: [TestComponent],
+            providers: [IfActiveService, IF_ACTIVE_ID_PROVIDER]
         });
         fixture = TestBed.createComponent(TestComponent);
         fixture.detectChanges();
         compiled = fixture.nativeElement;
     });
 
-    it("initializes the correct property values", () => {
-        let tabContents: TabContent[] = fixture.componentInstance.tabContentChildren.toArray();
-
-        expect(tabContents[0].active).toEqual(true);
-        expect(tabContents[1].active).toEqual(false);
+    it("creates ng-templates that can be rendered through templateOutlet", () => {
+        let tabContents: HTMLElement[] = compiled.querySelectorAll("clr-tab-content");
+        expect(tabContents.length).toEqual(2);
+        expect(tabContents[0].childNodes[0].nodeType).toEqual(Node.TEXT_NODE);
```

Comment:
This is a strange way to test it, I'm not sure what's happening. Why not just test `tabContent. templateRef`?